### PR TITLE
add Gardebook Controller

### PR DESCRIPTION
**Pull request by ricgarciacsumb**

### CODE REVIEW

**Comment by ricgarciacsumb** [comment]
@ddelaCSUMB

File: cst438_gradebook/src/test/java/com/cst438/JunitTestUpdateAssignmentName.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    	Assignment assign1 = new Assignment();
+    	AssignmentListDTO.AssignmentDTO assign = new AssignmentListDTO.AssignmentDTO(1, 0, null, null, null);
+		//Course course = new Course();
+		assign.courseId = 2;
+		//course.getAssignments().add(assign);
+		assign.assignmentName ="Assignment 2";
+        
+        given(assignmentRepository.findById(1)).willReturn(Optional.of(assign1));
+        
+//        Assignment savedAssignment = new Assignment();
```
Comment (ddelaCSUMB) [inline cst438_gradebook/src/test/java/com/cst438/JunitTestUpdateAssignmentName.java:75]: commented out lines could be deleted, also indentation is a little all over the place

File: cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import com.cst438.domain.AssignmentListDTO;
+import com.cst438.domain.AssignmentRepository;
+import com.cst438.domain.Course;
+import com.cst438.domain.CourseRepository;
+
+//import java.util.Arraylist;
+
+@RestController 
+public class AssignmentController 
+{
```
Comment (ddelaCSUMB) [inline cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java:20]: assignment controller looks good on indentation and commented well.

Comment (ddelaCSUMB) [reply cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java:20]: only suggestion I can think of is maybe look into java Optional objects to handle null pointer exceptions a little better, overall looks great!

**Review by ddelaCSUMB — COMMENTED** [review]
lookin good, Junit test could be cleaned up a little with unused lines of code and indentation.

**Pull request merged**